resource-watch/resource-watch
# next@9

## Overview
Upgrades RW to `next@9`

## Testing instructions
Run `yarn` to install the new version. Run the server as usual.
Run `rm -rf node_modules && yarn` for double check.

## Pivotal task
https://www.pivotaltracker.com/story/show/167161535

---

## Checklist before submitting
